components/scream/src/physics/p3/atmosphere_microphysics.cpp

```diff
@@ -94,6 +94,83 @@ void P3Microphysics::set_grids(const std::shared_ptr<const GridsManager> grids_m
 
 }
 
+// =========================================================================================
+int P3Microphysics::requested_buffer_size_in_bytes() const
+{
+  const Int nk_pack = ekat::npack<Spack>(m_num_levs);
+
+  // Number of Reals needed by local views in the interface
+  const int interface_request =
+      // 1d view scalar, size (ncol)
+      Buffer::num_1d_scalar*m_num_cols*sizeof(Real) +
+      // 2d view packed, size (ncol, nlev_packs)
+      Buffer::num_2d_vector*m_num_cols*nk_pack*sizeof(Spack) +
+      // 2d view scalar, size (ncol, 3)
+      m_num_cols*3*sizeof(Real);
```

Comment:
So there are 3 2D scalars? Is it worth adding a `Buffer::num_2d_scalar` to reflect this? Or am I misunderstanding?